kong/tools/sandbox_helpers.lua

```diff
@@ -0,0 +1,193 @@
+-- XXX Use sandbox.lua once it gets published
+-- https://github.com/kikito/sandbox.lua/pull/2
+local _sandbox = require "kong.tools.sandbox"
+
+local utils = require "kong.tools.utils"
+
+local fmt = string.format
+local unpack = table.unpack
```

Comment:
```suggestion
local unpack = unpack or table.unpack
```

Reply:
Actually, I just changed this to use `string.match` instead of `split` and then `unpack` because the former two are notoriously slow. `split` allocates a new table and `unpack` is [NYI in LuaJIT](http://wiki.luajit.org/NYI#libraries_table-library).